src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixExtension.groovy

```diff
@@ -37,7 +39,10 @@ class ScalafixExtension {
 
     ScalafixExtension(Project project) {
         this.project = project
-        configFile = project.objects.fileProperty()
+        configFile = project.objects.fileProperty().convention(
+                project.getLayout().getProjectDirectory().file(DEFAULT_CONFIG_FILE).asFile.exists()?
+                        project.getLayout().getProjectDirectory().file(DEFAULT_CONFIG_FILE) :
+                        project.rootProject.getLayout().getProjectDirectory().file(DEFAULT_CONFIG_FILE))
```

Comment:
if the file does not exist in the root project either, we should return `null` instead of a path to a file that does not exist. Inside the task it gets wrapped in a `java.util.Optional`